### PR TITLE
Add support for Minio

### PR DESCRIPTION
**Pull request by masipcat**
* Added support for Minio (https://minio.io/)
* Replaced `'TRAVIS' in os.environ` with `IS_TRAVIS`

### CODE REVIEW

File: pytest_docker_fixtures/__init__.py [hunk 1]
```diff
@@ -72,3 +73,17 @@ def rabbitmq():
 def kafka():
     yield kafka_image.run()
     kafka_image.stop()
+
+
+@pytest.fixture(scope='session')
+def minio():
+    if IS_TRAVIS:
+        host = 'localhost'
+        port = 6379
```
Comment (vangheem) [inline pytest_docker_fixtures/__init__.py]: port `6379` for minio? Isn't that redis?

Does travis have minio support?

Comment (masipcat) [reply pytest_docker_fixtures/__init__.py]: AFAIK travis doesn't support Minio but we can run minio on `before_script` (maybe I'm wrong)

Comment (masipcat) [reply pytest_docker_fixtures/__init__.py]: > port 6379 for minio? Isn't that redis?
I forgot to change this line

**Comment by vangheem** [comment]
1.2.8 released

**Comment by masipcat** [comment]
thanks :)

**Pull request merged**